v63002/inref/gtm8760.m

```diff
@@ -0,0 +1,38 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+longstring
+	set x="longstring.txt"
+	open x
+	use x
+	for i=1:1:$zcmdlne do
```

Comment:
zcmdlne -> zcmdline

Also use the following line instead of lines 17-18. This is a lot faster.
```
write $translate($justify("a",+$zcmdline)," ","a")
```